example/src/main.jsx

```diff
@@ -393,3 +393,34 @@ function UsingTheAPI() {
 
 const container5 = ReactDOM.createRoot(document.getElementById('demo-5'))
 container5.render(<UsingTheAPI />)
+
+/**
+ * Demo 6 - Extending Tag type
+ */
+
+const _suggestions = [
```

Comment:
```suggestion
type MySuggestions = Array<TagSuggestion & { test: string }>

const _suggestions: MySuggestions = [
```

Looking at this now, is it an issue for implementors to extend the base `TagSuggestion` like so? The new type overlaps with the libraries internal type so I think it should work most of the time.